### PR TITLE
Docs:  file parameter import_tasks and include_tasks

### PR DESCRIPTION
**Pull request by oraNod**
##### SUMMARY
Fixes issue #79022 by adding the `file:` parameter to the import_tasks module and revising the description for the `file:` parameter in the include_tasks module.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
import_tasks and include_tasks

##### ADDITIONAL INFORMATION
See issue #79022

### CODE REVIEW

File: lib/ansible/modules/include_tasks.py [hunk 1]
```diff
@@ -18,9 +18,7 @@
 options:
   file:
     description:
-      - The name of the imported file is specified directly without any other option.
-      - Unlike M(ansible.builtin.import_tasks), most keywords, including loop, with_items, and conditionals, apply to this statement.
-      - The do until loop is not supported on M(ansible.builtin.include_tasks).
```
Comment (sivel) [inline lib/ansible/modules/include_tasks.py:23]: These lines should probably be moved to `notes:` and not just deleted.

Comment (oraNod) [reply lib/ansible/modules/include_tasks.py:23]: Thanks for catching that. I think I just interpreted things the wrong way. I've added these lines back in and tried to tidy them up so they match the details in the import_tasks module docs.

File: lib/ansible/modules/import_tasks.py [hunk 2]
```diff
@@ -18,9 +18,14 @@
 options:
   free-form:
     description:
-      - The name of the imported file is specified directly without any other option.
-      - Most keywords, including loops and conditionals, only applied to the imported tasks, not to this statement itself.
+      - Specifies the name of the imported file directly without any other option C(- import_tasks: file.yml).
```
Comment (felixfontein) [inline lib/ansible/modules/import_tasks.py]: Thanks to the `: ` the YAML parser things that you are defining a dictionary in this list item. Adding quotes ixes that.
```suggestion
      - "Specifies the name of the imported file directly without any other option C(- import_tasks: file.yml)."
```

File: lib/ansible/modules/include_tasks.py [hunk 3]
```diff
@@ -30,9 +28,10 @@
     version_added: '2.7'
   free-form:
     description:
-      - |
-        Supplying a file name via free-form C(- include_tasks: file.yml) of a file to be included is the equivalent
-        of specifying an argument of I(file).
+      - Specifies the name of the imported file directly without any other option C(- include_tasks: file.yml).
```
Comment (felixfontein) [inline lib/ansible/modules/include_tasks.py]:
```suggestion
      - "Specifies the name of the imported file directly without any other option C(- include_tasks: file.yml)."
```

**Comment by oraNod** [comment]
@felixfontein It looks like CI is failing because: lib/ansible/modules/import_tasks.py:0:0: option-incorrect-version-added: version_added for new option (file) should be '2.15'. Currently StrictVersion ('2.7')

However 2.15 is not the version when the file option was actually added. Should I maybe just remove `version_added: '2.7'`?

Thanks.

**Comment by felixfontein** [comment]
> @felixfontein It looks like CI is failing because: lib/ansible/modules/import_tasks.py:0:0: option-incorrect-version-added: version_added for new option (file) should be '2.15'. Currently StrictVersion ('2.7')
> 
> However 2.15 is not the version when the file option was actually added. Should I maybe just remove `version_added: '2.7'`?

You can simply ignore that error; this is one of the errors that is only flagged in PRs, and this is one of the situations where that error is a false positive. (It's mainly a protection that in case someone actually adds a new parameter, they use the correct version added value. This test always fails when you later on add documentation that should have been added in a previous version...)

**Comment by webknjaz** [comment]
@mattclay looks like we may need somebody to "force-merge" this because of a corner case. Do we have any mechanism for one-off ignores?

**Comment by bcoca** [comment]
either add ignore for the test or merge with current version and do followup to update to older version

**Comment by felixfontein** [comment]
> either add ignore for the test or merge with current version and do followup to update to older version

You can't ignore this one by adding ignore.txt entries, since the test is only running during PR CI. Once it is merged, the test no longer runs, and thus the ignore would break CI.

**Comment by samccann** [comment]
@oraNod - not sure this would backport cleanly to 2.14 with my batch of docs backports this week. If you feel it should be in stable-2.14 please create a backport for it (so I can duck and hide from whatever CI problems popped up here :-)
